cores/esp8266/PolledTimeout.h

```diff
@@ -237,10 +248,16 @@ class timeoutTemplate
   }
   
   IRAM_ATTR // fast
-  bool expiredOneShot() const
+  bool expiredOneShot()
   {
     // returns "always expired" or "has expired"
-    return !canWait() || checkExpired(TimePolicyT::time());
+    return !canWait();
+    if (checkExpired(TimePolicyT::time()))
```

Comment:
unreachable code ?

Reply:
LOL - certainly so in C++. Thanks.

Reply:
For some fun, this is supposedly valid in Python:
```
return A+1 if A > B else A-1
```